tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -273,12 +273,6 @@ ROOT::Experimental::RNTupleFillContext::RNTupleFillContext(std::unique_ptr<ROOT:
                                                            std::unique_ptr<ROOT::Experimental::Detail::RPageSink> sink)
    : fSink(std::move(sink)), fModel(std::move(model)), fMetrics("RNTupleFillContext")
 {
-   if (!fModel) {
-      throw RException(R__FAIL("null model"));
-   }
```

Comment:
same here, we should keep the null model check because it's provided by the user